viewer/packages/tools/src/Measurement/styles/Label.css

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css?family=Inter&display=swap');
```

Comment:
Not familiar with this, is it done runtime? Is it possible to have some sort of logic like this:
- If font Inter is available use that,
- else use a standard available font.

Such that we don't have to load the font.

Reply:
Actually, the font-family css key is already setup with this in mind, just add a fall-back and everything should be fine.